src/app/components/home/home.component.html

```diff
@@ -1,16 +1,234 @@
+          </div>
+          <div class="col s12 m2 l2">
+            <img src="../../../assets/images/features/faster-eval.png" alt="" width="100%">
+          </div>
+          <div class="col s12 m4 l4">
+            <h5 class="w-300">Faster evaluation</h5>
+            <p class="w-300 fs-16">We warm-up the worker nodes at start-up by importing the challenge code and pre-loading the dataset in memory. We also split the dataset into small chunks that are simultaneously evaluated on multiple cores. These simple tricks result in faster evaluation and reduces the evaluation time by an order of magnitude in some cases.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="ev-container ev-reverse-details text-med-black " id="ev-scroll-down">
+      <div class="grad-container text-med-black ">
+        <div class="row">
+          <h4 class="w-300 center">Popular challenges</h4>
+        </div>
+        <div class="row">
+          <div class="col s12 m3 l3" *ngFor="let challenge of challengeList">
+            <a class="ev-card-hover" [routerLink] = "['challenge', challenge['id']]">
+              <div class="card ev-card-panel ev-challenge-card ev-card-hover">
+                <div class="card-image ev-card-image">
+                  <img class="bg-img" [src]="challenge['image']">
+                  <span class=" ev-card-title fs-14"><span><img [src]="challenge['image']" ></span>{{challenge['image']}}</span>
+                </div>
+                <div class="card-content">
+                  <p><strong class="text-light-black fs-12">Organized by</strong>
+                    <br>
+                    <span class="w-300 fs-16">{{challenge['creator']['team_name']}}</span>
+                  </p>
+                  <p><strong class="text-light-black fs-12">Starts on</strong>
+                    <br>
+                    <span class="w-300 fs-16">{{challenge['start_date'] | date:'medium'}}</span>
+                  </p>
+                  <p><strong class="text-light-black fs-12">Ends on</strong>
+                    <br>
+                    <span class="w-300 fs-16">{{challenge['end_date'] | date:'medium'}}</span>
+                  </p>
+                </div>
+                <div class=" btn-card-detail waves-effect waves-dark w-300 fs-14"> <strong>View Details </strong> &nbsp; </div>
+              </div>
+            </a>
+          </div>
+          <div *ngIf="challengeList.length == 0" class="row">
+            <p class="text-dark-black w-300 center fs-16">Sorry! there are no featured challenges. Please check again later.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- organizatons section -->
+    <section class="ev-container ev-reverse-details text-med-black ev-super-light-bg " id="ev-scroll-down">
```

Comment:
Keep the ID's unique. This, https://github.com/Cloud-CV/EvalAI-ngx/pull/140/files#diff-52b8edd82669758b98abb08f9c2a93b1R212, and https://github.com/Cloud-CV/EvalAI-ngx/pull/140/files#diff-52b8edd82669758b98abb08f9c2a93b1R26 have the same ID's.

Reply:
done